### ISSUE TITLE
https://socialamnesia.com Let's Encrypt certificate expired in 2020

### ISSUE THREAD

**Issue by xzenor** [Nick-Gottschlich/Social-Amnesia#89]
I'm not sure where else to report this.. But the certificate on https://socialamnesia.com is expired.

**Comment by lucanomax** [comment]
I think this project is not mantained anymore, the website certificate is the last problem XD

**Comment by Nick-Gottschlich** [comment]
Yeah, I haven't had time to work on this in a long time. I will look into the cert when I get a chance.
